packages-native/intro-screen/webpack.config.dev.js

```diff
@@ -0,0 +1,10 @@
+const merge = require("webpack-merge");
+const baseConfig = require("../../configs/webpack.dev.native.js");
+
+const customConfig = {
+    resolve: {
+        extensions: [".js", ".jsx", ".ts", ".tsx", ".ios.js", ".ios.jsx", ".ios.ts", ".ios.tsx"]
```

Comment:
Why do we need ` ".ios.js", ".ios.jsx", ".ios.ts", ".ios.tsx"` ?

Reply:
Old library issues, just removed the custom configs